server/routes/routerRequestHandlers.ts

```diff
@@ -123,8 +123,14 @@ const retrievePrisonerSummaryIfNotInSession = (prisonerSearchService: PrisonerSe
           prisonNumber: prisoner.prisonerNumber,
           prisonId: prisoner.prisonId,
           releaseDate: prisoner.releaseDate,
-          firstName: `${prisoner.firstName.trim().charAt(0).toUpperCase()}${prisoner.firstName.trim().slice(1)}`,
-          lastName: `${prisoner.lastName.trim().charAt(0).toUpperCase()}${prisoner.lastName.trim().slice(1)}`,
+          firstName: `${prisoner.firstName.trim().charAt(0).toUpperCase()}${prisoner.firstName
+            .trim()
+            .slice(1)
+            .toLowerCase()}`,
+          lastName: `${prisoner.lastName.trim().charAt(0).toUpperCase()}${prisoner.lastName
+            .trim()
+            .slice(1)
+            .toLowerCase()}`,
```

Comment:
Took a similar approach to https://github.com/mozilla/nunjucks/blob/master/nunjucks/src/filters.js#L50-L54 here but I think this one is a little easier to read?

Reply:
What about a small private function?
```
const capitalize = (name: string): string => {
  const trimmedLowercaseName = name.trim().toLowerCase()
  return trimmedLowercaseName.charAt(0).toUpperCase() + trimmedLowercaseName.slice(1)
```

Reply:
Makes sense!

Reply:
Added